### ISSUE TITLE
"cannot find macro `log!` in this scope" when compiling with log 0.4.1

### ISSUE THREAD

**Issue by kaedroho** [alexheretic/glyph-brush#51]
I'm getting the following error when compiling ``glyph_brush`` in my project.

```
error: cannot find macro `log!` in this scope
   --> C:\Users\karl\.cargo\registry\src\github.com-1ecc6299db9ec823\glyph_brush-0.2.4\src\glyph_brush.rs:250:33
    |
250 |                                 error!("Cache miss?: {:?}, {:?}: {}", font_id, glyph, err); 
    |                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info

error: aborting due to previous error
error: Could not compile `glyph_brush`.
```

The versions it's trying to compile are:

 - ``glyph_brush`` 0.2.4
 - ``log`` 0.4.1

rustc version:  ``rustc 1.31.1 (b6c32da9b 2018-12-18)`` (stable)
OS: Windows 10

Also (in case it's worth noting), in my project, the ``glyph_brush`` and ``log`` are pulled in as dependencies of``ggez`` (which I'm currently updating to the latest git revision ``c6d9ee4389f14502abd6a313001daaf30e7173e1``).

**Comment by kaedroho** [comment]
Manually bumping ``log`` to 0.4.6 fixes this issue, so probably worth pinning it in ``Cargo.toml``?

**Comment by alexheretic** [comment]
Yeah this was a macro exporting issue in older log, we can & should bump the min to `0.4.6` (or whatever version this was addressed) in glyph-brush's Cargo.toml as you suggest. Would you like to raise a pr?

**Comment by alexheretic** [comment]
This was addressed in log `0.4.4` so I'll use that as a minimum. I'll make a release at the end of the coming week. For anyone suffering from this run `cargo update`.
